pkg/services/object/get/v2/util.go

```diff
@@ -70,7 +71,7 @@ func (s *Service) toPrm(req *objectV2.GetRequest, stream objectSvc.GetObjectStre
 		var onceHeaderSending sync.Once
 		var globalProgress int
 
-		p.SetRequestForwarder(groupAddressRequestForwarder(func(addr network.Address, c client.MultiAddressClient, pubkey []byte) (*object.Object, error) {
+		p.SetRequestForwarder(groupAddressRequestForwarder(func(ctx context.Context, addr network.Address, c client.MultiAddressClient, pubkey []byte) (*object.Object, error) {
```

Comment:
lets use it then (same for `toRangePrm`)

Reply:
do you mean `context`? HASH is the only request that is not a stream so it does not have its own `context` when `forwarded` is called

Reply:
dont mind, added a commit about that